blocks/identity-block/utils/validate-redirect-url.js

```diff
@@ -2,9 +2,22 @@ const validateURL = (url) => {
 	if (!url) return null;
 	const validationRegEx = /^\/[^/].*$/;
 	const valid = validationRegEx.test(url);
+
 	if (valid) {
+		// handling preview with PB.
+		if (window.location.pathname.includes('/pf/')) {
```

Comment:
Hi @Bill, Everything looks good, but do you mind to check if there is another way (existing method) to know if we are in preview mode instead of check the URL contains `/pf/`